libnd4j/include/system/op_boilerplate.h

```diff
@@ -2385,6 +2385,7 @@
     return sd::ops::LogicOp::validateAndExecute(block);                     \
   };
 
+//TODO: each op implementation needs a new op registrator register call. It needs to be defined in this template.
```

Comment:
Is this todo taken care of?

Reply:
Yeah it is. This turned out to be false. Instead of each template the DecalrableOp itself just needed the call instead.